browser/js/login/createUserAccount.html

```diff
@@ -0,0 +1,60 @@
+<div class="col-md-4 col-md-offset-4">
+	<div class="panel panel-default">
+		<div class="panel-heading">Create User</div>
+		<div class="panel-body">
+			<form name="createUserAccount" ng-submit="createUser(user)">
```

Comment:
Change to ng-submit="createUserAccount.$valid && createUser(user)" to only allow submission when form is valid